powrap/powrap.py

```diff
@@ -1,53 +1,45 @@
 #!/usr/bin/env python3
 
-"""Fix style of uncommited po files, or all if --all is given.
-"""
+"""Fix style of uncommited po files, or all if --all is given."""
 
 import argparse
+import math
 import sys
 from typing import Iterable
 from pathlib import Path
-from subprocess import check_output, run
-from tempfile import NamedTemporaryFile
+from subprocess import check_output
 
+import polib
 from tqdm import tqdm
 
 from powrap import __version__
 
 
-def check_style(po_files: Iterable[str], no_wrap=False, quiet=False):
-    """Check style of given po_files
-    """
+def check_style(po_files: Iterable[str], no_wrap: bool = False, quiet: bool = False):
+    """Check style of given po_files."""
+    _pofile_kwargs = {}
+    if no_wrap:
+        _pofile_kwargs["wrapwidth"] = math.inf
```

Comment:
Should you set explicitly the 79 width in an else clause ?

Reply:
Yes, it's safer. But polib declares it as `78`.